compile aria2(1.31.0) fails

```
arm-unknown-linux-uclibcgnueabi-gcc (crosstool-NG crosstool-ng-1.22.0-dirty) 5.2.0
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

```

```
export CROSS2=arm-unknown-linux-uclibcgnueabi
./configure --host $CROSS2 --target=$CROSS2
configure: summary of build options:

Build:          x86_64-pc-linux-gnu
Host:           arm-unknown-linux-uclibcgnueabi
Target:         arm-unknown-linux-uclibcgnueabi
Install prefix: /usr/local
CC:             arm-unknown-linux-uclibcgnueabi-gcc
CXX:            arm-unknown-linux-uclibcgnueabi-g++
CPP:            arm-unknown-linux-uclibcgnueabi-gcc -E
CXXFLAGS:       -g -O2
CFLAGS:         -g -O2
CPPFLAGS:       
LDFLAGS:        
LIBS:           
DEFS:           -DHAVE_CONFIG_H
CXX1XCXXFLAGS:   -std=c++11 
EXTRACXXFLAGS:   -pipe
EXTRACFLAGS:     -pipe
EXTRACPPFLAGS:  
EXTRALDFLAGS:   
EXTRALIBS:      
WARNCXXFLAGS:   
LibUV:          no (CFLAGS='' LIBS='')
SQLite3:        no (CFLAGS='' LIBS='')
SSL Support:    yes
AppleTLS:       no (LDFLAGS='')
WinTLS:         no (LIBS='')
GnuTLS:         no (CFLAGS='' LIBS='')
OpenSSL:        yes (CFLAGS='' LIBS='-lssl -lcrypto')
CA Bundle:      
LibNettle:      no (CFLAGS='' LIBS='')
LibGmp:         no (CFLAGS='' LIBS='')
LibGcrypt:      no (CFLAGS='' LIBS='')
LibXML2:        no (CFLAGS='' LIBS='')
LibExpat:       no (CFLAGS='' LIBS='')
LibCares:       no (CFLAGS='' LIBS='')
Zlib:           yes (CFLAGS='' LIBS='-lz')
Libssh2:        no (CFLAGS='' LIBS='')
Tcmalloc:       no (CFLAGS='' LIBS='')
Jemalloc:       no (CFLAGS='' LIBS='')
Epoll:          yes
Bittorrent:     yes
Metalink:       no
XML-RPC:        no
Message Digest: openssl
WebSocket:      yes (CFLAGS='-I$(top_builddir)/deps/wslay/lib/includes -I$(top_srcdir)/deps/wslay/lib/includes' LIBS='$(top_builddir)/deps/wslay/lib/libwslay.la')
Libaria2:       no (shared=yes static=no)
bash_completion dir: ${datarootdir}/doc/${PACKAGE_TARNAME}/bash_completion
Static build:   no

```

```
make
SocketCore.cc: In member function 'void aria2::SocketCore::applyIpDscp()':
SocketCore.cc:587:32: error: 'IPV6_TCLASS' was not declared in this scope
       setSockOpt(IPPROTO_IPV6, IPV6_TCLASS, &ipDscp_, sizeof(ipDscp_));
                                ^
Makefile:2315: recipe for target 'SocketCore.lo' failed
```
help me with cross compile

--- a/src/SocketCore.cc
+++ b/src/SocketCore.cc
@@ -581,8 +581,8 @@ void SocketCore::applyIpDscp()
     if (family == AF_INET) {
       setSockOpt(IPPROTO_IP, IP_TOS, &ipDscp_, sizeof(ipDscp_));
     }
-#if defined(__linux__) || defined(__FreeBSD__) || defined(__NetBSD__) ||       \
-    defined(__OpenBSD__) || defined(__DragonFly__)
+#if defined(IPV6_TCLASS) || defined(__linux__) || defined(__FreeBSD__) ||      \
+    defined(__NetBSD__) || defined(__OpenBSD__) || defined(__DragonFly__)
     else if (family == AF_INET6) {
       setSockOpt(IPPROTO_IPV6, IPV6_TCLASS, &ipDscp_, sizeof(ipDscp_));
     }

